domains/estebanrg.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "estebanrg",
+    "email": "[ricardoespace@gmail.com]"
```

gh username and invalid email
```suggestion
    "username": "esteban-rg"
```